CK3toEU4/Source/CK3World/Dynasties/Dynasties.cpp

```diff
@@ -20,8 +20,13 @@ void CK3::Dynasties::registerKeys()
 		dynasties = Dynasties(theStream).getDynasties();
 	});
 	registerRegex(R"(\d+)", [this](const std::string& gameID, std::istream& theStream) {
-		const auto newDynasty = std::make_shared<Dynasty>(theStream, std::stoi(gameID));
-		dynasties.insert(std::pair(newDynasty->getGameID(), newDynasty));
+		const auto suspiciousItem = commonItems::singleItem(gameID, theStream);
+		if (suspiciousItem.find('{') != std::string::npos)
```

Comment:
You should add a test for whatever nonsense you're correcting for here.
It should fail on whatever the code was before and succeed with this change.